### PR TITLE
Photonize wordmark

### LINKED ISSUE #2971: Apply Photon Wordmark
https://drive.google.com/open?id=0B-WT7MpBRN0NNS03M0RxdTBHUlk

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #2971 

DO NOT MERGE UNTIL AFTER Milestone 34 is tagged!

### CODE REVIEW

File: locales/en-US/app.ftl [hunk 1]
```diff
@@ -268,3 +268,6 @@ warningMissingPrefDetail=When running Test Pilot locally or in development envir
 warningBadHostnameTitle=Unapproved hostname!
 warningBadHostnameDetail=The Test Pilot site may only be accessed from testpilot.firefox.com, testpilot.stage.mozaws.net, testpilot.dev.mozaws.net, or example.com:8000. Please see <a>our documentation</a> for details.
 
+// This string does not appear in app, but we will use it to localize our `no script` message
+jsDisabledWarning = Test Pilot requires JavaScript. Sorry about that.
```
Comment (flodolo) [inline locales/en-US/app.ftl]: Looks like this one sneaked in from another PR

**Comment by johngruen** [comment]
@lmorchard this should be ready to go now

**Comment by meandavejustice** [comment]
This seems to introduce an issue at smaller resolutions where part of the Firefox logo is repeating and being cut off to the right of the text.
BEFORE:
2017-10-31--10-17-23-1920x1080_scrot
AFTER:
2017-10-31--09-17-53-1920x1080_scrot

File: frontend/src/styles/modules/_main-header.scss [hunk 2]
```diff
@@ -5,15 +5,16 @@
...
       @include respond-to('not-small') {
-        @include hidpi-background-image('wordmark', 296px 64px);
+        @include hidpi-background-image('wordmark', 302px 64px);
         height: 64px;
-        width: 296px;
+        width: 302px;
       }
-      @include hidpi-background-image('wordmark', 190px 41px);
+      @include hidpi-background-image('wordmark', 194px 41px);
+      background-repeat: none;
```
Comment (meandavejustice) [inline frontend/src/styles/modules/_main-header.scss]: Seems weird that the logo seems to be still showing up again in the lower resolution(mobile) view when `background-repeat` is set here.. hmmm

**Review by meandavejustice — CHANGES_REQUESTED** [review]

**Comment by johngruen** [comment]
@meandavejustice put `none` instead of `no-repeat`

**Comment by lmorchard** [comment]
Looks :100: to me now

**Pull request merged**